src/udev/udev-builtin-net_id.c

```diff
@@ -80,6 +80,7 @@ typedef struct LinkInfo {
         int ifindex;
         int iflink;
         int iftype;
+        int vf_repr_num;
```

Comment:
all this stuff is cryptic enough. can we maybe for once not abbreviate so badly. i.e. if "repr" means "representator", then maybe spell that out? There's no shame in using slightly longer identifiers.

Reply:
OK